bin/cron/admin_stats

```diff
@@ -4,47 +4,105 @@
+  # ...for measuring the percentage of students that started thirty-seven to forty-four days ago
+  #    and logged in at least once after thirty days.
+  thirty_day_retention_current_weekly_baseline = students.
+    where(created_at: (Time.now - (THIRTY_DAYS + 2 * SEVEN_DAYS))..(Time.now - (THIRTY_DAYS + SEVEN_DAYS)))
+  thirty_day_retention_current_weekly = thirty_day_retention_current_weekly_baseline.
+    where("current_sign_in_at - created_at < ?", %q{interval '30 days'})
+  # ...for measuring the percentage of students that started sixty to ninety days ago and logged in
+  #    at least once after thirty days
+  thirty_day_retention_current_monthly_baseline = students.
+    where(created_at: (Time.now - (3 * THIRTY_DAYS))..(Time.now - (2 * THIRTY_DAYS)))
+  thirty_day_retention_current_monthly = thirty_day_retention_current_monthly_baseline.
+    where("current_sign_in_at - created_at < ?", %q{interval '30 days'})
+
+  # Students and teachers.
   followers = DASHBOARD_REPORTING_DB_READONLY[:followers]
-  prizes = DASHBOARD_REPORTING_DB_READONLY[:prizes]
-  teacher_prizes = DASHBOARD_REPORTING_DB_READONLY[:teacher_prizes]
-  teacher_bonus_prizes = DASHBOARD_REPORTING_DB_READONLY[:teacher_bonus_prizes]
 
   Properties.set(:admin_stats, {
-    user_count: users.count,
-    teacher_count: teachers.count,
-    student_count: students.count,
-    users_with_teachers: followers.distinct.count(:student_user_id),
-    users_with_email: users.where('email <> ""').count,
-    users_with_confirmed_email: users.where('confirmed_at IS NOT NULL').count,
-    girls: students.where(:gender => 'f').count,
-    boys: students.where(:gender => 'm').count,
-
-    prizes_redeemed: prizes.where('user_id IS NOT NULL').group(:prize_provider_id).count,
-    prizes_available: prizes.where('user_id IS NULL').group(:prize_provider_id).count,
-
-    student_prizes_earned: users.where(:prize_earned => true).count,
-    student_prizes_redeemed: prizes.where('user_id IS NOT NULL').count,
-    student_prizes_available: prizes.where('user_id IS NULL').count,
-
-    teacher_prizes_earned: users.where(:teacher_prize_earned => true).count,
-    teacher_prizes_redeemed: teacher_prizes.where('user_id IS NOT NULL').count,
-    teacher_prizes_available: teacher_prizes.where('user_id IS NULL').count,
-
-    teacher_bonus_prizes_earned: users.where(:teacher_bonus_prize_earned => true).count,
-    teacher_bonus_prizes_redeemed: teacher_bonus_prizes.where('user_id IS NOT NULL').count,
-    teacher_bonus_prizes_available: teacher_bonus_prizes.where('user_id IS NULL').count,
+    accounts: {
+      users: all_users.count,
+      students: all_students.count,
+      teachers: all_teachers.count,
+    },
+    born_accounts: {
+      users: users.count,
+      students: students.count,
+      female_students: students.where(gender: 'f').count,
+      male_students: students.where(gender: 'm').count,
+      teachers: teachers.count,
+      female_teachers: teachers.where(gender: 'f').count,
```

Comment:
Teacher gender is probably not very interesting because we don't ask teachers their gender on sign up: https://studio.code.org/users/sign_up?user%5Buser_type%5D=teacher

(if you see some in the DB could be from, 1) it's possible that we did in the past, 2) people can create student accounts then change them to teacher, 3) we can get gender from oauth)